src/Service/CLI/Commands/MigrateCommand.php

```
@@ -0,0 +1,152 @@
+     *      $ wp statistics migrate --batch-size=500 --yes
+     *
+     *      # Force re-migration
+     *      $ wp statistics migrate --force --yes
+     *
+     * @subcommand v14-to-v15
+     *
+     * @param array $args       Positional arguments.
+     * @param array $assoc_args Associative arguments.
+     * @return void
+     */
+    public function v14_to_v15($args, $assoc_args)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$args'. (undefined)

(UnusedFormalParameter)